easybuild/easyblocks/generic/intelbase.py

```diff
@@ -215,9 +215,19 @@ def prepare_step(self, requires_runtime_license=True):
         self.requires_runtime_license = self.cfg['requires_runtime_license'] and requires_runtime_license
 
         if self.requires_runtime_license:
+            license_file = self.cfg['license_file']
+            if isinstance(license_file, basestring):
+                license_specs = [license_file]
+            elif isinstance(license_file, list):
```

Comment:
W.r.t. `tuple`, better check whether `find_flexlm_license` would handle that correctly (I think it would, but I'm not sure)